src/Libraries/CoreNodes/FileSystem.cs

```diff
@@ -117,11 +118,13 @@ public static bool FileExists(string path)
         /// </summary>
         /// <param name="filePath">Path to write to</param>
         /// <param name="text">Text content</param>
+        /// <returns name="success"> Success message if file was written correctly </returns>
         /// <search>write file,text,file,filepath</search>
-        public static void WriteText(string filePath, string text)
+        public static string WriteText(string filePath, string text)
         {
             var fullpath = AbsolutePath(filePath);
             System.IO.File.WriteAllText(fullpath, text);
+            return "File has been succesfully written";
```

Comment:
This is an API break. Please undo this change.

Reply:
Addressed in a0e7b13